autoPyTorch/pipeline/components/training/trainer/mixup_utils.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Callable, Dict, Optional
+
+from ConfigSpace.conditions import EqualsCondition
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    Constant,
+    UniformFloatHyperparameter,
+)
+
+import numpy as np
+
+from sklearn.utils import check_random_state
+
+from autoPyTorch.constants import CLASSIFICATION_TASKS, STRING_TO_TASK_TYPES
+from autoPyTorch.pipeline.components.training.trainer.utils import Lookahead
+from autoPyTorch.utils.common import HyperparameterSearchSpace, add_hyperparameter, get_hyperparameter
+
+
+class MixUp:
```

Comment:
As mentioned in the `cutout` class, I do not need the necessity of the separation of those classes and we can just use `**kwargs`.